test/common/pom.xml

```diff
@@ -97,5 +101,23 @@
       <groupId>org.skyscreamer</groupId>
       <artifactId>jsonassert</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>testcontainers</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>elasticsearch</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>elasticsearch</artifactId>
+      <scope>compile</scope>
```

Comment:
I'm still not sure about including these deps twice in this way. It's very odd 😅. I will try to figure out whether it's possible to leave out one of the two.

Reply:
Yes, this is very strange, but I was not able to build it without this.